packages/contracts/contracts/L1/SecurityCouncil.sol

```diff
@@ -0,0 +1,139 @@
+ *         and allows security council parties to validate & agree on transactions before execution.
+ */
+contract SecurityCouncil is MultiSigWallet, Semver {
+    /**
+     * @notice Represents an l2 output data.
+     *
+     * @custom:field outputRoot     Root of the L2 output root that was proven against.
+     * @custom:field l2BlockNumber  L2 block number that the output corresponds to.
+     */
+    struct OutputInfo {
+        bytes32 outputRoot;
+        uint128 l2BlockNumber;
+    }
+
+    /**
+     * @notice A mapping of output info array which has been submitted to validate.
+     */
+    mapping(uint256 => OutputInfo) public outputInfos;
+
+    /**
+     * @notice The address of the colosseum contract. Can be updated via upgrade.
+     */
+    address public immutable COLOSSEUM;
+
+    /**
+     * @notice Emitted when a validation request is submitted.
+     *
+     * @param transactionId Index of the submitted transaction.
+     * @param outputRoot    L2 output root that was proven against.
+     * @param l2BlockNumber The L2 block number of the output root.
+     */
+    event Validation(uint256 indexed transactionId, bytes32 outputRoot, uint128 l2BlockNumber);
+
+    /**
+     * @notice Disallow calls from anyone except Colosseum.
+     */
+    modifier onlyColosseum() {
+        require(
+            msg.sender == COLOSSEUM,
+            "SecurityCouncil: only the colosseum contract can be a sender"
+        );
+        _;
+    }
+
+    /**
+     * @custom:semver 0.1.0
+     *
+     * @param _colosseum Address of the Colosseum contract.
+     */
+    constructor(address _colosseum) Semver(0, 1, 0) {
+        COLOSSEUM = _colosseum;
+    }
+
+    /**
+     * @notice Initializer.
+     *
+     * @param ``                        Not used. Dummy parameter to prevent override.
+     * @param _owners                   List of initial owners.
+     * @param _numConfirmationsRequired Number of required confirmations.
+     *
+     */
+    function initialize(
+        bool,
+        address[] memory _owners,
+        uint256 _numConfirmationsRequired
+    ) public initializer {
+        MultiSigWallet.initialize(_owners, _numConfirmationsRequired);
+    }
+
+    /**
+     * @notice Allows the Colosseum to request for validate output data.
+     *
+     * @param _outputRoot    Output root byte data.
+     * @param _l2BlockNumber L2 block number corresponding to output.
+     * @param _destination   Address that calldata call to.
+     * @param _data          Calldata for callback purpose.
+     *
+     * @return Returns transaction ID validated.
+     */
+    function requestValidation(
+        bytes32 _outputRoot,
+        uint128 _l2BlockNumber,
+        address _destination,
+        bytes memory _data
+    ) public onlyColosseum returns (uint256) {
+        outputInfos[transactionCount] = OutputInfo({
+            outputRoot: _outputRoot,
+            l2BlockNumber: _l2BlockNumber
+        });
```

Comment:
As you said, a map based on transactionCount doesn't look good. 
Instead of adding a separate counter, it would make sense to store it based on transactionId. 
That way, we don't have to have a separate view function as well.

The output index values are managed at the outputOracle level, so I don't want to add them here.